### PR TITLE
Add no.mifi.losslesscut

### PR DESCRIPTION
**Pull request by muelli**
https://github.com/mifi/lossless-cut

I have upstreamed the patches, so they won't be needed for the next release, I hope.
https://github.com/mifi/lossless-cut/pull/233

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build no.mifi.losslesscut

**Comment by muelli** [comment]
hm. that's weird. it builds locally. I run `flatpak-builder --user --install --ccache --keep-build-dirs  --force-clean -v  --repo=/var/tmp/fb.repo fpbuilder no.mifi.losslesscut.yaml ` with flatpak-builder 1.0.9.
How can it build fine locally but not on the infrastructure?

**Comment by p1u3o** [comment]
```
FB: Running: flatpak build --die-with-parent --env=FLATPAK_BUILDER_BUILDDIR=/run/build/losless-cut --nofilesystem=host --filesystem=/srv/buildbot/worker/build-x86_64-6/build/.flatpak-builder/build/losless-cut-1 --bind-mount=/run/build/losless-cut=/srv/buildbot/worker/build-x86_64-6/build/.flatpak-builder/build/losless-cut-1 --build-dir=/run/build/losless-cut --bind-mount=/run/ccache=/srv/buildbot/worker/build-x86_64-6/build/.flatpak-builder/ccache --unshare=network --env=SOURCE_DATE_EPOCH=1576448531 '--env=CFLAGS=-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection' '--env=CXXFLAGS=-O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection' '--env=LDFLAGS=-L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed' --env=ELECTRON_CACHE=/run/build/losless-cut/flatpak-node/electron-cache --env=electron_config_cache=/run/build/losless-cut/flatpak-node/electron-cache --env=npm_config_cache=/run/build/losless-cut/flatpak-node/npm-cache/ --env=npm_config_nodedir=/usr/lib/sdk/node10 --env=NPM_CONFIG_LOGLEVEL=info --env=CCACHE_DIR=/run/ccache/disabled --env=PATH=/app/bin:/usr/bin:/usr/lib/sdk/node10/bin --env=LD_LIBRARY_PATH=/app/lib --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig --env=FLATPAK_BUILDER_N_JOBS=48 /srv/buildbot/worker/build-x86_64-6/build/.flatpak-builder/rofiles/rofiles-qIwdQt /bin/sh -c 'yarn run pack-linux dir'
yarn run v1.19.1
warning You don't appear to have an internet connection. Try the --offline flag to use the cache for registry queries.
error Couldn't find a package.json file in "/run/build/losless-cut"
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.```

The build servers don't allow network access during the build process.

**Comment by muelli** [comment]
ah. no. generated-sources was missing. How could the build have gotten that far if the sources were missing?

**Comment by muelli** [comment]
bot, build no.mifi.losslesscut

**Comment by LeandroStanger** [comment]
bot, build no.mifi.losslesscut

File: no.mifi.losslesscut.yaml [hunk 1]
```diff
@@ -0,0 +1,135 @@
+app-id: no.mifi.losslesscut
+# I have no idea, actually, what a base does and whether we need it.
```
Comment (barthalion) [inline no.mifi.losslesscut.yaml]: You don't need for building from source.

**Comment by muelli** [comment]
bot, build no.mifi.losslesscut

**Comment by muelli** [comment]
@bilelmoussaoui you've set the "waiting for upstream" tag. What are we waiting for?

**Comment by mifi** [comment]
Do I need to do something?

**Comment by bilelmoussaoui** [comment]
There's nothing to do other than updating the app manifest once there's a new release. The app somehow didn't build once merged. I have triggered a new build here https://flathub.org/builds/#/builders/32/builds/17629. It should appear on Flathub in few hours

**Pull request closed without merging**